src/User/UserContainerRedirector.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { useCookies } from "react-cookie";
+import { cookieName } from "../Constants/Cookies";
+import { Redirect } from "react-router-dom";
+import UserContainer from "./UserContainer";
+
+const UserContainerRedirector = () => {
+  const [cookies] = useCookies([cookieName]);
+
+  return (
+    <>
+      {cookies.sessionToken ? (
+        <UserContainer />
+      ) : (
+        <Redirect to="/users/login" />
+      )}
+    </>
+  );
+};
```

Comment:
make generic?

`
const UserRedirector = ({redirectPath, children}) => {
  const [cookies] = useCookies([cookieName]);

  return (
      cookies.sessionToken ? children : <Redirect to={redirectPath} />
`